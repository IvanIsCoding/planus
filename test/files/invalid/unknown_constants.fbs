table Foo {
  x: float32 = NaN;
  y: float32 = hello;
}