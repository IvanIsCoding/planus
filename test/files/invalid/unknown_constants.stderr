[0m[1m[38;5;9merror[0m[1m: Unknown constant "hello"[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/unknown_constants.fbs:3:16
  [0m[34mâ”‚[0m
[0m[34m3[0m [0m[34mâ”‚[0m   y: float32 = [0m[31mhello[0m;
  [0m[34mâ”‚[0m                [0m[31m^^^^^[0m

Error: Bailing because of errors
